next-seo.config.js

```diff
@@ -0,0 +1,33 @@
+const WEB_DEFAULT_TITLE = '🇮🇩 INA COVID BED';
+const WEB_DEFAULT_DESCRIPTION = 'Informasi ketersediaan tempat tidur di Rumah Sakit di seluruh Provinsi di Indonesia.';
+const WEB_DEFAULT_URL = 'https://ina-covid-bed-tau.vercel.app/';
+
+export default ({
```

Comment:
Could we change this to named function? Like this:
```js
export default function SEO() { ... }
```

Or it's should be anonymous function?

Reply:
hmm.. the one that shouldn't be anonymous is react-component right? I think this would not affect performance ga.